src/components/Send.jsx

```diff
@@ -286,7 +296,19 @@ export default function Send() {
               title="Send bitcoin"
               subtitle="Collaborative transactions increase the privacy of yourself and others."
             />
+
+            <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
+              <div className="mb-4 border border-1 p-3" style={{ borderRadius: '.25rem' }}>
+                🛈{` `}
+                <small className="text-secondary">
+                  {makerRunning && <>Earn is active. Stop the service in order to send collaborative transactions.</>}
+                  {coinjoinInProcess && <>A collaborative transaction is in progress.</>}
```

Comment:
```suggestion
                  {coinjoinInProcess && <>A collaborative transaction is currently in progress.</>}
```

Reply:
Okay, had that in the previous version (see the phrasing history, ha!) but thought it sounds kind of strange when the user _just_ initiated the transaction. Will add "currently" (also matches the wording on the Earn page)!